### DISCUSSION TITLE
Can not start with secure boot enabled 1600x900 display

### DISCUSSION THREAD

**Discussion by zJairO** [MultimediaLucario/Lenovo-ThinkPad-T440s#30] [Q&A] [answered]
Good afternoon, I have installed mac os big sur two weeks ago, everything is working fine, just I can not start with secure boot, I have default 1900x600 display, I get error when booting through secure boot, am I doing something wrong or have to do extra steps? Thanks.

Opencore ver 0.9.0

![image](https://user-images.githubusercontent.com/5152239/229379899-57769121-716b-417d-92f6-004006cc4d18.jpeg)

![Screen Shot 2023-04-08 at 23 34 49](https://user-images.githubusercontent.com/5152239/230756271-c01da659-4623-4cb7-968a-ee6e8fe78c99.png)

**Comment by MultimediaLucario** [comment, marked as answer]
I recommend disabling Secure Boot in the BIOS.
